family2family/src/test/java/org/cbtf/f2f/findamentor/FirstPotentialMentorControllerIntegrationTest.java

```diff
@@ -0,0 +1,65 @@
+package org.cbtf.f2f.findamentor;
+
+import org.cbtf.f2f.Family2FamilyApp;
+import org.cbtf.f2f.common.F2FMediaTypes;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+import static org.hamcrest.Matchers.is;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * This integration class is for testing the {@link FirstPotentialMentorController}. It relies on data that is loaded
+ * from the data-dev.sql file.
+ */
+@RunWith(SpringRunner.class)
+@SpringBootTest(classes = Family2FamilyApp.class)
+@WebAppConfiguration
+public class FirstPotentialMentorControllerIntegrationTest {
```

Comment:
Pretty nice way to test rest controllers. Can depend on the data loaded by the data-dev.sql file and use MockMvc to make requests and verify responses. Once an API is determined, this would be a good way for QA to test out different scenarios. :-D